File: kumoh-code-2017/Java/#1 기본/#1 과제/#5 java 과제/ppt 초본/은행관리/BalanceInquiry.java
public class BalanceInquiry extends Transaction
{
	public BalanceInquiry(int account_number, Screen screen, BankDatabase bank_database)
	{
		super(account_number, screen, bank_database);
	}
	
	public void execute()
	{
		BankDatabase bank_database = getBankDatabase();
		Screen screen = getScreen();
		
		int balance = bank_database.getBalance(getAccountNumber());
		
		screen.displayMessage("¿‹∞Ì : ");
		screen.displayAmount(balance);
		screen.displayMsgLine("");
	}
}

File: kumoh-code-2017/C++/#2 설계/과제15 은행관리 프로그램/BalanceInquiry.cpp
#include"BalanceInquiry.h"

BalanceInquiry::BalanceInquiry(int accountNumber, Screen screen, BankDatabase bankDB) :Transaction( accountNumber,  screen,  bankDB){

}

void BalanceInquiry::execute(){
	BankDatabase bankDatabase = getBankDataBalance();
	Screen screen = getScreen();
	int balance = bankDatabase.getBalance(getAccountNumber());
	screen.displayMsg("¿‹∞Ì: ");
	screen.displayAmount(balance);
	screen.displayMsg("");
}